src/client/api.ts

```diff
@@ -403,6 +405,16 @@ export async function fetchOrganizationFeaturedProjects(
   });
 }
 
+export const fetchMemoizedStateBbox = memoize(
+  async (region: IRegionConfig): Promise<IStaticMetadata["bbox"]> => {
+    const staticMetadata = await fetchStaticMetadata(region.s3URI);
+    return staticMetadata.bbox;
+  },
+  {
+    normalizer: (args: [region: IRegionConfig]) => JSON.stringify(args[0].name)
```

Comment:
`RegionConfig.name` is not required to be unique (https://github.com/PublicMapping/districtbuilder/blob/334331cddc6d2c9860699917fc02ada3ccbd4a74/src/server/src/region-configs/entities/region-config.entity.ts#L22-L23) so we shouldn't rely on that for the `memoize` cache key. Using `id` should work though:
```suggestion
    normalizer: (args: [region: IRegionConfig]) => JSON.stringify(args[0].id)
```